src/frontend/screens/Game/ModifyInstallModal/index.scss

```diff
@@ -45,7 +45,7 @@
     }
   }
   &__branchPassword {
-    width: 30vw;
+    width: 30vw !important;
```

Comment:
![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/30195425/ba0566d5-09ec-470d-9e60-b60b118170d5)

It's still too wide and overflows the box, should be set to be within the dimensions of the dialog box.

Reply:
I believe I fixed it in previous commit

Reply:
Hm, at the time I tested it out it was like that, I just installed the latest flatpak on this branch and it worked without an issue. Can be resolved.